tests/scripts/all.sh

```diff
@@ -6065,6 +6065,11 @@ component_test_suite_with_psasim()
 
     helper_psasim_server start
 
+    # psasim takes an extremely long execution time on some test suites so we
+    # exclude them from the list.
+    SKIP_TEST_SUITES="constant_time_hmac,lmots,lms"
+    export SKIP_TEST_SUITES
```

Comment:
Won't this mean that `SKIP_TEST_SUITES` will skip these tests in any components that run after this one? Would it be better to set it on the `make` command line instead?

Reply:
It's fine. Components run in a subshell so that their state doesn't risk contaminating subsequent components. (Except for leaving a file behind that `make clean` doesn't remove, or for changing source files other than the ones that are backed up.)